data/release/global.xml

```diff
@@ -15,7 +15,20 @@
     <onlineResource>http://geoserver.org</onlineResource>
     <verbose>false</verbose>
     <verboseExceptions>false</verboseExceptions>
+    <metadata>
+      <map>
+        <entry>
+          <string>CogSettings.Key</string>
```

Comment:
@aaime did you intend to include COG settings as part of this change?

Reply:
I did not.

Reply:
However looking at the diff, I don't see a change in globa.xml?
The current global.xml does not contain that metadata entry either:

https://github.com/geoserver/geoserver/blob/main/data/release/global.xml

Not sure why you're pointing at this particular PR?

Reply:
Ah, if you look at the history, the change has been reverted in the same day. Again, what brought you to look at this particular PR?